src/utils/DateUtils.js

```diff
@@ -20,9 +20,15 @@ const getWeekdaysMin = () => {
 
 const getVisibleDays = (month, year) => {
   const offset = moment([year, month]).startOf('month').weekday();
-  const previousMonthLength = daysInMonthCount(month - 1, year) + 1; //We need the last number too
   const currentMonthLength = daysInMonthCount(month, year) + 1;
 
+  if (month - 1 < 0) {
+    month = 12;
```

Comment:
why did you change this from `11` to `12`? aren't the months going from `0` to `11`?